src/seedu/addressbook/data/person/Address.java

```diff
@@ -3,29 +3,26 @@
 import seedu.addressbook.data.exception.IllegalValueException;
 
 /**
+ * Subclass Contact
```

Comment:
Phrase too short to make sense. In any case, the parent class is specified in the class declaration, so no need to mention here.